akka-projection-core/src/main/scala/akka/projection/MergeableOffset.scala

```diff
@@ -4,4 +4,6 @@
 
 package akka.projection
 
+import akka.annotation.ApiMayChange
+@ApiMayChange
```

Comment:
[minor] Line break? Not sure if this is something the formatter should have done, doesn't seem to matter.